### PR TITLE
[styles][Refactor] added modal to menu drawer

### PR DESCRIPTION
**Pull request by abhij1607**
added modal for side menu drawer in mobile screen

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
The feature works as expected, needs some design improvements in future refactor.

Good to merge for now.

File: src/components/NavBar/nav-components/top-nav.js [hunk 1]
```diff
@@ -32,52 +32,55 @@ const TopNav = () => {
...
-      >
-        <div className="drawer-header pd-y-base pd-x-base flex-row">
-          <div className="drawer-title">
-            <i className="fa fa-2x fa-user" aria-hidden="true"></i>
+
+      <div className={`modal-wrapper fade ${isSideDrawerActive ? "show" : ""}`}>
+        <aside
+          id="side-nav"
+          className={`sidebar-nav modal-nav ${
+            isSideDrawerActive ? "active" : ""
```
Comment (saadahmadhere) [inline src/components/NavBar/nav-components/top-nav.js:40]: could consider using && operator here.

**Review by saadahmadhere — COMMENTED** [review]
Nicely written code. Made some suggestions. Everything else looks good.

File: src/components/NavBar/nav-components/top-nav.js (hunk 1 above)
Comment (abhij1607) [reply src/components/NavBar/nav-components/top-nav.js:40]: but it'll add false in class names

**Pull request merged**